go/vt/servenv/pprof_test.go

```diff
@@ -45,7 +45,7 @@ func TestParseProfileFlag(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.arg, func(t *testing.T) {
-			got, err := parseProfileFlag(tt.arg)
+			got, err := parseProfileFlag(strings.Split(tt.arg, ","))
```

Comment:
`TestParseProfileFlag` fails for the subtest with `""` as profile now because `len(strings.Split("", ","))` is 1 and not 0 ...